nuage_metroae_config/actions.py

```diff
@@ -6,6 +6,9 @@
                     TemplateParseError)
 from logger import Logger
 from util import get_dict_field_no_case
+from os import remove
+import zipfile
+import base64
```

Comment:
Is base64 a standard library?  Does it need to be added to requirements.txt?

Reply:
It is a standard library